docs/source/grid/args-loaders.rst

```diff
@@ -35,6 +35,12 @@ applies session information as needed.
 .. autoclass:: webgrid.extensions.RequestArgsLoader
     :members:
 
+.. autoclass:: webgrid.extensions.RequestFormLoader

```

Comment:
We'll need an edit in the section above this to reflect that the default args are a blank MultiDict now, rather than request URL args.